libs/ui-toolkit/src/components/dropdown-menu/dropdown-menu-grid.tsx

```diff
@@ -0,0 +1,31 @@
+import * as DropdownMenuPrimitive from '@radix-ui/react-dropdown-menu';
+import classNames from 'classnames';
+import { forwardRef } from 'react';
+import { Icon } from '../icon';
+
+/**
+ * The button that toggles the dropdown menu inside grids.
+ * By default, the {@link DropdownMenuContent} will position itself against the trigger.
+ */
+export const DropdownMenuGridTrigger = forwardRef<
```

Comment:
chore (blocking): The icon that you choose to render is the apps responsibility, we shouldn't add components to handle this, what happens when someone else needs a dropdown with a different icon? Are we just going to add dropdown triggers for every icon? I would move this to the app

Reply:
moved the trigger to the app in accounts container